src/renderer/components/TransactionConfirm/index.js

```diff
@@ -91,18 +111,18 @@ const TransactionConfirm = ({ t, device, account, parentAccount, transaction, st
             />
           </TransactionConfirmField>
         )}
-
-        <TransactionConfirmField label={<Trans i18nKey="send.steps.details.fees" />}>
-          <FormattedVal
-            color={"palette.text.shade80"}
-            unit={feesUnit}
-            val={estimatedFees}
-            fontSize={3}
-            inline
-            showCode
-          />
-        </TransactionConfirmField>
-
+        {!estimatedFees.isZero() && (
```

Comment:
I'm not sure about this. Can you confirm that the fees does not appear at all on the device in that case as well?

Reply:
I can confirm they don't appear yes :+1: